docs/themes/oso-tailwind/layouts/partials/sidebar.html

```diff
@@ -27,7 +27,7 @@
                 {{ $currentNode := . }}
                 {{ if eq $currentNode.Section "new-roles" }}
                 <ul>
-                    <li><a href="{{ .Site.Home.Permalink }}">Back to documentation home</a></li>
+                    <li><a href="/">Back to documentation home</a></li>
```

Comment:
```suggestion
                    <li><a href="{{ .Site.Home.Permalink }index.html">Back to documentation home</a></li>
```

alternate

Reply:
what's the benefit?

Reply:
another: https://github.com/osohq/oso/blob/e1f274fdda65c988201ba8a76a3937a99a2da327/docs/themes/oso-tailwind/layouts/partials/header.html#L16

Reply:
I think my version would redirect back to the same language index.